docs/en/user_guides/config.md

```diff
@@ -1 +1,709 @@
+    metrics=dict(),
+    dataset_prefixes=['IC13', 'IC15'])
+
+# Multiple datasets, multiple Metric
+val_evaluator = dict(
+    type='MultiDatasetsEvaluator',
+    metrics=[...],
+    dataset_prefixes=['IC13', 'IC15'])
+```
+
+#### Metric
+
+Metrics refer to different measures of accuracy, while multiple metrics can be used together, for more metrics principles refer to {external+mmengine:doc}`MMEngine: Metric <tutorials/evaluation>`, there are different metrics for different algorithmic tasks in MMOCR. More OCR-related  metrics can be found in [Evaluation](../basic_concepts/evaluation.md).
+
+Text detection: [`HmeanIOUMetric`](mmocr.evaluation.metrics.HmeanIOUMetric)
+
+Text recognition: [`WordMetric`](mmocr.evaluation.metrics.WordMetric), [`CharMetric`](mmocr.evaluation.metrics.CharMetric), [`OneMinusNEDMetric`](mmocr.evaluation.metrics.OneMinusNEDMetric)
+
+Key information extraction: `F1Metric`
+
+Text detection as an example, using a single `Metric` in the case of single dataset evaluation.
+
+```Python
+val_evaluator = dict(type='HmeanIOUMetric')
+```
+
+Taking text recognition as an example, multiple datasets are evaluated using multiple `Metric`.
+
+```Python
+val_evaluator = dict(
+    type='MultiDatasetsEvaluator',
+    metrics=[
+        dict(
+            type='WordMetric',
+            mode=['exact', 'ignore_case', 'ignore_case_symbol']),
+        dict(type='CharMetric')
+    ],
+    dataset_prefixes=['IC13', 'IC15'])
+test_evaluator = val_evaluator
+```
+
+<div id="vis_config"></div>
+
+### Visualizaiton Configuration
+
+Each task is configured with a visualizer corresponding to that task. The visualizer is mainly used for visualizing or storing intermediate results of user models and visualizing val and test prediction results. The visualization results can also be stored in different backends such as WandB, TensorBoard, etc. through the visualization backend. Commonly used modification operations can be found in [visualization](visualization.md).
+
+The default configuration of visualization for text detection is as follows.
+
+```Python
+vis_backends = [dict(type='LocalVisBackend')]
+visualizer = dict(
+    type='TextDetLocalVisualizer',  # Different visualizers for different tasks
+    vis_backends=vis_backends,
+    name='visualizer')
+```
+
+## Directory Structure
+
+All configuration files of `MMOCR` are placed under the `configs` folder. To avoid long configuration files and to improve the reusability and clarity of configuration files, MMOCR takes advantage of the inheritance property of Config files to split the eight sections of configuration content. Since each section is related to an algorithm task, MMOCR provides a task folder for each task in Config, namely `textdet` (text detection task), `textrecog` (text recognition task), and `kie` (key information extraction). Also the individual task algorithm configuration folder is further divided into two parts: `_base_` folder and many algorithm folders.
```

Comment:
```suggestion
All configuration files of `MMOCR` are placed under the `configs` folder. To avoid config files from being too long and improve their reusability and clarity, MMOCR takes advantage of the inheritance mechanism and split config files into eight sections. Since each section is closely related to the task type, MMOCR provides a task folder for each task in `configs/`, namely `textdet` (text detection task), `textrecog` (text recognition task), and `kie` (key information extraction). Each folder is further divided into two parts: `_base_` folder and algorithm configuration folders.
```